src/tsconfig-base.json

```diff
@@ -22,6 +22,7 @@
 
         "alwaysStrict": true,
         "preserveConstEnums": true,
+        "downlevelIteration": true,

```

Comment:
#32221 tracks what we'd need to do to target native generators (mostly fixing TDZ bugs to do with real runtime `const`/`let`), which is pretty appealing at this point, since all versions of `node` we support use them, and they consume 1/3rd of the memory.

Reply:
`"downlevelIterator"` will seriously degrade the performance of all of our `for..of` statements in the compiler. Using native generators would also be a problem for VSCode/Monaco as they still have web hosting scenarios they need to support where native generators are not available.

Reply:
Oh yeah, I know - that's why I did testing with native generators (and other es6 features). Native generators are fine for most people, though, and we can always just ship a secondary bundle with downleveled generators for older webhosts.